recipes/usockets/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+import os
+
+from conans import ConanFile, tools, MSBuild
+from conans.errors import ConanInvalidConfiguration
+
+
+class UsocketsConan(ConanFile):
+    name = "usockets"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Miniscule cross-platform eventing, networking & crypto for async applications"
+    license = "Apache-2.0"
+    homepage = "https://github.com/uNetworking/uSockets"
+    topics = ("conan", "socket", "network", "web")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"fPIC": [True, False],
+               "with_openssl": [True, False]}
+    default_options = {"fPIC": True, 'with_openssl': False}
+    generators = "cmake"
+    _source_subfolder = "source_subfolder"
+
+    def config_options(self):
+        # no shared, no fPIC at all
+        del self.options.fPIC
```

Comment:
That's not correct: a static library can still be used inside a shared .so library.
```suggestion
        if self.settings.os == "Windows":
            del self.options.fPIC
```